src/net.cpp

```diff
@@ -2539,6 +2539,18 @@ size_t CConnman::GetNodeCount(NumConnections flags)
     return nNum;
 }
 
+bool CConnman::GetNodeStats(NodeId nodeid, CNodeStats& stats)
```

Comment:
This method would also be removed if fetching CNode was done at the updatepeer level.